cypress-tests/cypress/e2e/PaymentUtils/Stripe.js

```diff
@@ -126,6 +165,29 @@ export const connectorDetails = {
         status: 200,
         body: {
           status: "requires_capture",
+          payment_method: "card",
+          attempt_count: 1,
+          payment_method_data: {
+            card: {
+              last4: "4242",
+              card_type: "CREDIT",
+              card_network: "Visa",
+              card_issuer: "STRIPE PAYMENTS UK LIMITED",
+              card_issuing_country: "UNITEDKINGDOM",
+              card_isin: "424242",
+              card_extended_bin: null,
+              card_exp_month: "10",
+              card_exp_year: "25",
+              card_holder_name: null,
+              payment_checks: {
+                cvc_check: "pass",
+                address_line1_check: "pass",
+                address_postal_code_check: "pass",
+              },
+              authentication_data: null,
+            },
+            billing: null,
+          },
```

Comment:
i guess this can be an exception as payment_checks has different data compared to the other 2:

```js
payment_checks: {
  cvc_check: "pass",
  address_line1_check: "pass",
  address_postal_code_check: "pass",
},
```